src/styles/search-results.css

```diff
@@ -0,0 +1,26 @@
+p {
+  text-align: center;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.image-container {
+  margin: 1rem;
```

Comment:
A note about units here, its best to use rem for font sizes, em for things like margin and padding, and px for small things like border widths. Think Ive mentioned this to you already, but if not have a read of this https://dev.to/theodorusclarence/back-to-basic-should-we-use-rem-em-or-pixel-1hd0